src/api/uploads/MultiputPart.js

```diff
@@ -339,6 +345,48 @@ class MultiputPart extends BaseMultiput {
         this.destroy();
     }
 
+    /**
+     * Pauses upload for this Part.
+     *
+     * @return {void}
+     */
+    pause(): void {
+        this.isPaused = true;
+        this.state = PART_STATE_DIGEST_READY;
+        this.xhr.abort(); //  This calls the error handler.
```

Comment:
@cweeii When we go to upload a file, we use `xhr.uploadFile`. When user disconnects from the internet mid-upload, we still attempt to upload the parts for 5 min. Once we reach the timeout, in `xhr.uploadFile`, there is an `idleTimeoutFunc` that calls the first `abort` for one of the parts that was still uploading. `xhr.abort` results in cancelling the request (`axios.cancel`) for the part which results in an error. The error then gets handled by `partUploadErrorHandler` because when we first populate the parts in `MultiputUpload.populateParts` we set that to be the error handler. 

We want to abort on pause because if one part is aborted, we shouldn't let the rest of the parts just hang around or be cancelled because cancelling destroys the parts and we need to reuse the parts and their xhrs to resume. We have to reuse the parts because the API validates the parts using a unique sha.

Reply:
And yes u are right that it is an intended action when the user pauses their upload, which is why `isDestroyedOrPaused()` was created so that we ignore the errors that were caused from aborting through pause.